hack/csi_test/cmd/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	csi "sigs.k8s.io/gcp-filestore-csi-driver/hack/csi_test/pkg/csi_tool"
+)
+
+var (
+	endpoint    = flag.String("endpoint", "/tmp/csi.sock", "CSI endpoint")
+	host        = flag.String("host", "localhost", "Host machine name")
+	volume      = flag.String("volume", "", "Volume name")
```

Comment:
would "shareName" make it more obvious? This is more inline with what Filestore calls it

Reply:
Also changing this, thanks!